packages/bpk-scrim-utils/src/withScrim.js

```diff
@@ -38,6 +39,29 @@ const getClassName = cssModules(STYLES);
 
 const withScrim = WrappedComponent => {
   class WithScrim extends Component {
+    static propTypes = {
+      dark: PropTypes.bool,
+      getApplicationElement: PropTypes.func.isRequired,
```

Comment:
Should put the required one at the top to match convention.

Reply:
Yeah, good catch